src/redux/rockets/rocket.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const ROCKET_API = 'https://api.spacexdata.com/v3/rockets';
+const initialState = [];
+
+export const fetchRockets = createAsyncThunk('rocket/fetchRockets',
+  async () => {
+    const response = await axios.get(ROCKET_API)
+      .then((response) => response.data);
+    return response;
+  });
+
+const rocketSlice = createSlice({
+  name: 'rocket',
+  initialState,
+  extraReducers: (builder) => {
+    builder.addCase(fetchRockets.fulfilled, (state, action) => {
+      if (action.payload === '') {
+        return state;
+      }
+      return [...action.payload];
```

Comment:
##### Hello @ifeanyi305 here you are returning an array that contains all the properties from the response, but based on the requirement each object should only contain selected properties:

![image](https://user-images.githubusercontent.com/72081991/208739032-0dd75d43-6704-4634-838e-b8fc8731a3b0.png)
##### Kindly you can use this implementation as a reference: [example](https://github.com/yonas44/space-travelers-hub/pull/30/files#diff-294ef6a50d7479c57e21e641a40898bda7578fba0c54ad7a389df0209f057979R27-R34)